CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Features
 
+- Support for the Bitcoin canister has been added to `@dfinity/ckbtc` by exposing the function `getUtxos`.
```

Comment:
This is a bit confusing, how about "Support for `getUtxos` endpoint in the new Bitcoin canister from `@dfinity/ckbtc`"

Reply:
Thanks for the suggestion, that's way better! Updated.